src/main/docs/index.md

```diff
@@ -50,6 +50,17 @@ Here is the simplest way to connect, query and disconnect
 {@link examples.Examples#gettingStarted()}
 ```
 
+Environment variables can also be used to set default connection setting values, this is useful
```

Comment:
rather word it : "You can also use environment variables to set ..."

Reply:
Fixed in https://github.com/reactiverse/reactive-pg-client/pull/87/commits/1d086c389ecc5f521a9482fbe72ebede6625f81b.